cmd/channel/distributed/receiver/main.go

```diff
@@ -103,8 +111,13 @@ func main() {
 	}
 	defer kafkaProducer.Close()
 
+	var env envConfig
+	if err := envconfig.Process("", &env); err != nil {
+		logger.Fatal("Failed To Initialize env config", zap.Error(err))
+	}
+	channelReporter := eventingchannel.NewStatsReporter(env.ContainerName, kmeta.ChildName(env.PodName, uuid.New().String()))
 	// Create A New Knative Eventing MessageReceiver (Parses The Channel From The Host Header)
```

Comment:
This could be used in `handleMessage` to report errors and msg metrics (as with the in-memory channel case), although kafka producer/consumer do produce low level metrics.